packages/reason-relay/package.json

```diff
@@ -31,32 +31,32 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.4.0",
     "bs-fetch": "^0.5.0",
-    "bs-platform": "^7.1.1",
+    "bs-platform": "^7.3.2",
     "graphql": "14.5.0",
     "graphql-query-test-mock": "^0.12.1",
     "jest": "^24.9.0",
     "nock": "^11.7.0",
     "node-fetch": "^2.6.0",
-    "react": "0.0.0-experimental-d28bd2994",
-    "react-dom": "0.0.0-experimental-d28bd2994",
-    "react-relay": "0.0.0-experimental-8cc94ddc",
-    "reason-react": "^0.7.0",
-    "relay-compiler": "9.0.0",
-    "relay-config": "9.0.0",
-    "relay-runtime": "9.0.0",
-    "reason-promise": "^1.0.2"
+    "react": "^16.9.0",
+    "react-dom": "^16.9.0",
+    "reason-promise": "^1.0.2",
+    "reason-react": "^0.8.0",
+    "relay-compiler": "9.1.0",
+    "relay-config": "9.1.0",
+    "relay-experimental": "^0.10.0",
+    "relay-runtime": "9.1.0"
   },
   "peerDependencies": {
     "bs-platform": "^7.1.1",
     "graphql": "^14.4.2",
-    "react": "0.0.0-experimental-d28bd2994",
-    "react-dom": "0.0.0-experimental-d28bd2994",
-    "react-relay": "0.0.0-experimental-8cc94ddc",
```

Comment:
This needs to remain, and be upgraded to the latest `react-relay@experimental`. It also needs to stay pinned to a specific version.